ironfish/src/memPool/feeEstimator.ts

```diff
@@ -152,8 +179,13 @@ export class FeeEstimator {
     }
 
     fees.sort((a, b) => (a < b ? -1 : a > b ? 1 : 0))
+    const lastBlockSize = await this.getLastBlockSizeBytes()
+    const blockSizeRatio = lastBlockSize / this.wallet.chain.consensus.MAX_BLOCK_SIZE_BYTES
```

Comment:
It looks like the ratio here is only calculated from the size of the most recent block. That will be very sensitive to individual blocks -- if we have one empty block, then the fee rate will drop to zero.

Reply:
in geth implementation, it also only consider one parent block. If we really have a empty block(due to no transaction) then the next transaction should be the minimal since its extremely low demands.